lib/nxt_support/models/preprocess_attributes.rb

```diff
@@ -0,0 +1,34 @@
+module NxtSupport
+  module PreprocessAttributes
+    extend ActiveSupport::Concern
+    class_methods do
```

Comment:
There should be a newline between the `extend` and the `class_methods` imho.

```suggestion

    class_methods do
```